JavaCryptographicArchitecture/src/Mac.crysl

```diff
@@ -16,8 +16,7 @@ OBJECTS
 
 EVENTS
 	g1: getInstance(macAlgorithm);
-	g2: getInstance(macAlgorithm);
```

Comment:
@rakshitkr thanks for detecting this typo. What is meant in fact is: `g2: getInstance(macAlgorithm, _);`, in the case when a provider is used. So, it is not a redundant event. Besides correcting this line, would you please correct the same typo made in the BouncyCastle-JCA ruleset in the Mac rule [here](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/BouncyCastle-JCA/src/Mac.crysl#L19), since you have opened this PR? Thanks a lot!